modules/end-user-guide/partials/proc_troubleshooting-network-problems.adoc

```diff
@@ -3,70 +3,24 @@
-
-* Open additional ports for a specific web site.
-* Enable WebSockets on the proxy server.
-
-
-[id="enabling-the-websocket-protocol_{context}"]
-== Enabling the WebSocket protocol
-
-Enabling the WebSocket protocol is critical for the proper functionality of {prod-short} IDE.
-
-While the WebSocket protocol itself is unaware of proxy servers and firewalls, HTTP servers can share their default HTTP and HTTPS ports with a WebSocket server.
-
-* HTTP: port 80
-* HTTPS: port 433
-
-Some proxy servers operate with WebSockets by default, but others prevent WebSockets from working correctly, which causes the connection to fail.
-
-In some cases, the proxy server requires the additional configuration, and the specific proxy servers need an upgrade, which allows WebSockets support.
-
-
-[id="troubleshooting-websocket-secure-connections_{context}"]
-== Troubleshooting WebSocket Secure connections
-
-Secure WebSocket connections improve confidentiality and also reliability because they reduce the risk of interference by bad proxies. {prod-short} operates under WebSocket Secure connections by default and usually no action is required. Some customer's security policy blocks some aspects of the WebSocket protocol that causes problems with proper {prod-short} functionality. Those problems are however out of scope for {prod-short}  support and have to be solved by a network administrator.
-
-To troubleshoot a failing WebSocket Secure (WSS) connection, use the instructions in this section.
+This section describes how to prevent or resolve issues related to network policies. 
+{prod-short} requires the availability of the WebSocket Secure (WSS) connections.
+Secure WebSocket connections improve confidentiality and also reliability because they reduce the risk of interference by bad proxies.
 
 .Prerequisites
 
-* Using a supported web browser:
-+
---
+* The WebSocket Secure (WSS) connections on port 443 must be available on the network. Firewall and proxy may need additional configuration.
+* Use a supported web browser:
 ** Chrome
 ** Firefox
---
-+
-NOTE: Using an unsupported web browser causes a connection interruption, followed by a warning message.
 
 .Procedure
 
-. Check browser support:
-.. Check that the WebSocket protocol is enabled using a link:https://www.websocket.org/echo.html[realtime web test] in one of the supported browsers.
-+
-If the problem is not resolved, follow with the next step.
-
-. Check proxy servers and firewalls settings:
-.. Ask the system administrator to check if there is a proxy server or firewall that blocks WebSocket Secure (WSS) connections on port 443.
-+
-Possible required actions:
-+
-* Add an exception to the firewall.
-* Have the proxy intercept WebSocket connection.
+. Verify the browser supports the WebSocket protocol. See: link:https://www.google.com/search?q=websocket+test[Searching a websocket test].
 
-.Verification
+. Verify firewalls settings: WebSocket Secure (WSS) connections on port 443 must be available.
```

Comment:
Verify firewall settings: WebSocket Secure (WSS) connections on port 443 must be available.

Reply:
What is the difference?

Reply:
firewalls/firewall

Reply:
Plural form is intentional, as any corporate environment will have at lest 2 firewalls for redundancy purpose.